ch8/Homework_Problems.ipynb

```diff
@@ -7,14 +7,49 @@
     "## Problem 1\n",
     "\n",
     "Write a function called vectorize(func) that takes a function as its only positional argument and returns a new function that calls func multiple times. The new function should accept any number of positional arguments. If N arguments are passed to the new function, func is called N times with each of the values as a single argument. The new function should return a list of return values from each invocation of func.\n",
-    "To get a feel for its use, take a look at the following example:\n",
-    "\n",
+    "To get a feel for its use, take a look at the following example:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "```\n",
     "from math import sqrt\n",
     "multisqrt = vectorize(sqrt)\n",
     "multisqrt(4.0, 25.0, 1.0, 10.0)\n",
-    "[2.0, 5.0, 1.0, 3.1622776601683795]\n",
+    "# out --> [2.0, 5.0, 1.0, 3.1622776601683795]\n",
     "multisqrt(100.0, 9.0, 16.0)\n",
-    "[10.0, 3.0, 4.0]"
+    "# out --> [10.0, 3.0, 4.0]\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from test_util import testP1\n",
+    "from problem_1 import vectorize\n",
```

Comment:
Please put the file and the class (with a `pass`) in the directory so they're not running into an import error (see ch2 for example: when they start from scratch running the ch2 homework problems notebook, they immediately see test failures.).

<img width="1140" alt="Screen Shot 2021-01-08 at 8 53 53 AM" src="https://user-images.githubusercontent.com/5744164/104030089-767d5d80-5190-11eb-97f3-1f00611611f6.png">